clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -312,7 +312,12 @@ describe("Fides-js TCF", () => {
       cy.intercept("PATCH", `${API_URL}${FidesEndpointPaths.NOTICES_SERVED}`, {
         fixture: "consent/notices_served_tcf.json",
       }).as("patchNoticesServed");
-      cy.get("#fides-modal-link").click();
+
+      cy.get("div#fides-banner").within(() => {
```

Comment:
maybe a bit too late now, but could be worth wrapping this whole block in a function `openModalFromBanner` and calling that instead of this whole nested thing